src/test/java/com/mongodb/kafka/connect/source/MongoCopyDataManagerTest.java

```diff
@@ -138,6 +140,7 @@ void testAppliesTheExpectedPipelines() {
     when(mongoDatabase.getCollection(TEST_COLLECTION, RawBsonDocument.class))
         .thenReturn(mongoCollection);
     when(mongoCollection.aggregate(expectedPipeline)).thenReturn(aggregateIterable);
+    when(aggregateIterable.allowDiskUse(true)).thenReturn(aggregateIterable);
```

Comment:
[optional]

Since here and in all other similar places we don't actually care whether `true` or `false` is passed, let's use `ArgumentMatchers.anyBoolean` instead of `true`.

Alternatively, if we do want to mock only if the default (`true`) value is specified (this does not seem to be the case), then I propose to use `COPY_EXISTING_ALLOW_DISK_USE_DEFAULT` (has to be made package-access) instead of `true`.

Reply:
Done